libr/debug/p/native/w32.c

```diff
@@ -151,28 +154,21 @@ static int w32dbg_SeDebugPrivilege() {
 	return ret;
 }
 
-static void print_lasterr(const char *str) {
-	/* code from MSDN, :? */
-	LPWSTR pMessage = L"%1!*.*s! %4 %5!*s!";
-	DWORD_PTR pArgs[] = { (DWORD_PTR)4, (DWORD_PTR)2, (DWORD_PTR)L"Bill",  // %1!*.*s!
-		(DWORD_PTR)L"Bob",                                                // %4
-		(DWORD_PTR)6, (DWORD_PTR)L"Bill" };                               // %5!*s!
+static void print_lasterr(const char *caller, char *cause) {
 	WCHAR buffer[200];
 	char cbuffer[100];
-	if (!FormatMessage (FORMAT_MESSAGE_FROM_STRING |
```

Comment:
This is pretty much just an example copied from MSDN (doesnt even consider 'char *str').